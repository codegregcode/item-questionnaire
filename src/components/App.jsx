import React from 'react';
import { Route, Routes } from 'react-router-dom';
import Welcome from './Welcome';
import SignUp from './SignUp';
import Header from './Header';
import ItemList from './ItemList';
import '../styles/app.css';

const App = () => (
  <div className="background">
    <div className="App">
      <Header />
      <div className="content">
        <Routes>
          <Route path="/" element={<Welcome />} />
          <Route path="/sign-up" element={<SignUp />} />
          <Route path="brochure" element={<ItemList />} />
        </Routes>
      </div>
    </div>
  </div>
);

export default App;
