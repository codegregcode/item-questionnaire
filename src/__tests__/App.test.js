import { React, render, screen } from '@testing-library/react';
import App from '../components/App';

describe('App', () => {
  it('renders h1 title', () => {
    render(<App />);
    const h1Element = screen.getByText(/sell us your stuff/i);
    expect(h1Element).toBeInTheDocument();
  });
});
