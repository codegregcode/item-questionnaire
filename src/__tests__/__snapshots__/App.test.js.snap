// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<DocumentFragment>
  <div
    class="background"
  >
    <div
      class="App"
    >
      <div
        class="container"
        data-testid="svg-container"
      >
        <svg
          preserveAspectRatio="xMinYMin meet"
          viewBox="0 0 500 250"
        >
          <path
            d="M0,50 C150,100 350,0 500,50 L500,0 L0,0 Z"
          />
        </svg>
      </div>
      <div
        class="title"
      >
        <img
          alt="logo"
          src="logo.png"
        />
      </div>
      <div
        class="content"
      >
        <div>
           
          <p>
            Thank you for showing an interest in our service.
          </p>
          <p>
            Before we send you the postage labels we would like to ask you a few questions regarding the items you were wishing to sell
          </p>
          <a
            href="/sign-up"
          >
            Click here to get started
          </a>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
